.cloud-build/notebook-index-cloudbuild.yaml

```diff
@@ -0,0 +1,33 @@
+steps:
+  # Show the gcloud info and check if gcloud exists
+  - name: ${_PYTHON_IMAGE}
+    entrypoint: /bin/sh
+    args:
+    - -c
+    - gcloud config list --quiet
```

Comment:
This is just for debugging so you can remove it.

Reply:
removed